src/gar.lua

```diff
@@ -235,7 +223,7 @@ function gar.leaveNetwork(from, currentTimestamp, msgId)
 
 	-- update global state
 	GatewayRegistry[from] = gateway
-	return utils.deepCopy(gateway)
+	return gateway
```

Comment:
Might be risky. Double check that nothing downstream mutates it. And/or perhaps rename any references retained from calling this function "unsafeGateway"

Reply:
only callers are main.lua and epoch distributions - but I don't want to risk it anyway so reverted in [13ce11e](https://github.com/ar-io/ar-io-network-process/pull/363/commits/13ce11e368e0a11f95a70792edcb18f348651ef9)